index.fa.md

```diff
@@ -0,0 +1,2474 @@
+---
+title: نامه سرگشاده ای برای برکناری ریچارد ام. استالمن از همه مناصب رهبری
+description: ما ، امضا کنندگان ، خواستار استعفا کل هیئت FSF و برکناری ریچارد ام. استالمن از تمام مناصب رهبری هستیم. ما دیگر به رنج رفتارهای وی ، دادن نقش رهبری به او یا ادامه پذیرفتن او و ایدئولوژی آسیب رسان و خطرناک او ادامه نخواهیم داد. 
+image: /assets/og-image.png
+twitter:
+  card: summary_large_image
+share_text: من از این فراخوان برای کنار رفتن کل هیئت FSF و برکناری ریچارد ام. استالمن از سمت های رهبری حمایت می کنم. 
+---
+
+{% include share-links.html %}
+
```

Comment:
You can leave all but the title off the translated version.

Reply:
Ok, I remove all of these lines.